### DISCUSSION TITLE
Fibrosis by Percolation

### DISCUSSION THREAD

**Discussion by adasi11** [rsachetto/MonoAlg3D_C#43] [Q&A]
Hey, here Albert!

I am trying to model fibrosis by percolation, setting specific cells to inactive (grid_cell->active = false;). 
I am using [modify_domain].

But even with a very low percentage of percolation (5%), the simulation freezes without returning an error. When analyzing the Transmembrane Voltage values in the last time step computed, some cells present either very big (>100 mV) or very low values (-200mV), eventually yielding NaNs in the cellular model. 

This seems an issue with boundary conditions. Maybe we could discuss it in our monthly meeting?

Many thanks,
Albert

**Comment by rsachetto** [comment]
Hi Albert, 

The modify_domain function is used to modify the domain after a time threshold. Is that what you really want to do?
Can you send me (by email) your ini file? Maybe it is a bug in the modify domain function.

And sure, we can discuss it in the  monthly meeting.

My best,
Rafael Sachetto
